xtask/codegen/css.ungram

```diff
@@ -459,6 +460,154 @@ CssFontFaceAtRule =
 	'font-face'
 	block: CssBlock
 
+
+// <container-condition> = [ <container-name> ]? <container-query>
+// <container-name> = <custom-ident>
+// <container-query>     = not <query-in-parens>
+//                      | <query-in-parens> [ [ and <query-in-parens> ]* | [ or <query-in-parens> ]* ]
+//
+// <query-in-parens>     = ( <container-query> )
+//                       | ( <size-feature> )
+//                       | style( <style-query> )
+//                       | <general-enclosed>
+//
+// <style-query>         = not <style-in-parens>
+//                       | <style-in-parens> [ [ and <style-in-parens> ]* | [ or <style-in-parens> ]* ]
+//                       | <style-feature>
+//
+// <style-in-parens>     = ( <style-query> )
+//                       | ( <style-feature> )
+//                       | <general-enclosed>
+
+// <container-condition> = [ <container-name> ]? <container-query>
+CssContainerAtRule =
+	'container'
+	name: CssIdentifier?
+	query: AnyCssContainerQuery
+	block: CssBlock
+
+// <container-query>
+AnyCssContainerQuery =
+	CssContainerNotQuery
+	| AnyCssContainerCombinableQuery
+
+AnyCssContainerCombinableQuery =
```

Comment:
Could you explain what each node would represent, maybe with an example? The reason I am asking is because what we decide here would affect media queries too